test/Elastic.Apm.Azure.Storage.Tests/AzureBlobStorageDiagnosticListenerTests.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;

```

Comment:
There are some unused things in this file - could you please run code cleanup on this?

Update: also in `AzureFileShareStorageDiagnosticListenerTests.cs` and `AzureQueueStorageDiagnosticListenerTests,cs`, `BlobContainerScope.cs`.

Reply:
done!